src/test/java/org/fcrepo/sword/http/SWORDServiceResourceTest.java

```diff
@@ -13,24 +13,38 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.fcrepo.sword.integration;
+package org.fcrepo.sword.http;
 
+import org.apache.abdera.model.Service;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+
+import static org.junit.Assert.assertEquals;
 
 /**
  * @author claussni
- * @date 19.02.15.
+ * @date 18.02.15.
  */
 @RunWith(SpringJUnit4ClassRunner.class)
-@ContextConfiguration("/spring-test/test-container.xml")
-public class AbstractSWORDProviderIT {
+@ContextConfiguration("/spring/sword.xml")
+@WebAppConfiguration
+public class SWORDServiceResourceTest {
```

Comment:
The servlet container is being spun up for this test, so I would still consider it an "integration test", and hence keep the class name as `SWORDServiceResourceIT`.
Unit tests (ClassTest.java) usually take the form of creating a `new` object and exercising its methods.